WalletWasabi.Gui/Converters/ErrorDescriptorToBorderColorConverter.cs

```diff
@@ -11,31 +11,16 @@ namespace WalletWasabi.Gui.Converters
 {
 	public class ErrorDescriptorToBorderColorConverter : IValueConverter
 	{
-		private ErrorDescriptor UndefinedExceptionToErrorDescriptor(Exception ex)
-		{
-			var newErrDescMessage = ExceptionExtensions.ToTypeMessageString(ex);
-			return new ErrorDescriptor(ErrorSeverity.Warning, newErrDescMessage);
-		}
-
 		public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
 		{
-			var descriptors = new ErrorDescriptors();
+			var descriptors = ErrorDescriptors.Create();
 
 			if (value is IEnumerable<object> rawObj)
 			{
 				foreach (var error in rawObj.OfType<ErrorDescriptor>())
 				{
 					descriptors.Add(error);
 				}
```

Comment:
We cannot get exceptions here?

Reply:
No, exceptions pass through binding normally.

Its how binding system communicated, it was a mistake to intercept these exceptions, as they are part of Avalonia internal system.

Should a converter be responsible for creating validation errors?

The exceptions this was detecting were related to UI issues, and these are spit out of avalonias logger (debug mode only to let developers know about binding issues). They are all warnings and not errors.

In anycase any exception that passes through the binding system is categorially not a validation issue and should not be shown in the validation system.